src/Wpf.Ui/Wpf.Ui.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <ProjectGuid>{1ADC87D1-8963-4100-845A-18477824718E}</ProjectGuid>
-    <TargetFrameworks>net6.0-windows;net5.0-windows;netcoreapp3.1;net48;net47;net461</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;net45</TargetFrameworks>
```

Comment:
This change cannot be allowed, we want precompiled binaries for given frameworks. Besides, netcore3.1 has already reached EOL.